contractcourt/htlc_timeout_resolver.go

```diff
@@ -483,13 +483,20 @@ func (h *htlcTimeoutResolver) sweepSecondLevelTx() error {
 			h.broadcastHeight,
 		))
 	}
+
 	_, err := h.Sweeper.SweepInput(
 		inp,
 		sweep.Params{
 			Fee: sweep.FeeEstimateInfo{
 				ConfTarget: secondLevelConfTarget,
 			},
 			Force: true,
+
+			// TODO(yy): make this a user-config option.
+			Budget: btcutil.Amount(inp.SignDesc().Output.Value),
+
+			// TODO(yy): specify a no-deadline here.
+			// DeadlineHeight:
```

Comment:
the deadline is the incoming CLTV because if the outgoing claims via the success clause after this incoming CLTV, we can lose money

Reply:
Deferred this to a new PR.